Tests/Reader/converted_ADM_stellartable.dat

```diff
@@ -0,0 +1,2001 @@
+# r mass_density temp Ye specific_internal_energy velocity pressure adm_density adm_momentum S_adm Srr_adm
```

Comment:
This file and the other table should probably only be committed once, in the top-level `data` directory`